### ISSUE TITLE
'exps' vs 'exponents' in chebfun() call

### ISSUE THREAD

**Issue by nickhale** [chebfun/chebfun#1469]
One can supply `'exps'` to the Chebfun constructor:
```
>> f = chebfun(@sqrt, [0, 1], 'exps', [.5, 0])
f =
   chebfun column (1 smooth piece)
       interval       length     endpoint values   endpoint exponents
[       0,       1]        1         0        1         [0.5      0]  
epslevel = 1.000000e-14   vscale = 1
```
but not `'exponents'`:
```
>> f = chebfun(@sqrt, [0, 1], 'exponents', [.5, 0])
Warning: Unrecognized input preference 'exponents'. 
> In chebtech.techPref (line 72)
  In chebtech2 (line 88)
  In smoothfun.constructor (line 63)
  In onefun.constructor (line 71)
  In bndfun (line 103)
  In classicfun.constructor (line 100)
  In fun.constructor (line 51)
  In chebfun/constructor>getFun (line 292)
  In chebfun/constructorNconstructorNoSplit (line 110)
  In chebfun/constructor (line 63)
  In chebfun (line 215) 
Warning: Function not resolved using 65537 pts. Have you tried 'splitting on'? 
> In chebfun/constructorNconstructorNoSplit (line 120)
  In chebfun/constructor (line 63)
  In chebfun (line 215) 
f =
   chebfun column (1 smooth piece)
       interval       length     endpoint values  
[       0,       1]    65537   1.6e-16        1 
epslevel = 1.852400e-10   vscale = 1
```

Is this intentional?

**Comment by kuanxu** [comment]
@nickhale No, I think. We just simply adopted what we used in V4.

**Comment by nickhale** [comment]
It seems confusing. Is there any documentation about passing exponents via the chebfun constructor? It seems that at the moment it is simply quietly supported for backwards compatibility.

**Comment by trefethen** [comment]
There some discussion in sections 9.2 and 9.3 of the Guide.

**Comment by nickhale** [comment]
Is there a reason not to also allow `exponents`? I know we specifically changed the property of the field in `@singfun` from `.exps` to `.exponents` to make it more readable. One would expect this to be even more important for a user facing feature in the constructor.

I don't feel strongly about this, but it confused even me for a few minutes. If someone tells me this isn't an issue then I'll happily close the ticket.

**Comment by trefethen** [comment]
OK with me so long as `exps` remains.

**Comment by kuanxu** [comment]
Resolved by #1517.
